packages/story-editor/src/app/media/local/useContextValueProvider.js

```diff
@@ -205,7 +213,26 @@ export default function useContextValueProvider(reducerState, reducerActions) {
         removeBaseColorProcessing({ id });
       })();
     },
-    [setBaseColorProcessing, removeBaseColorProcessing, updateBaseColor]
+    [setBaseColorProcessing, updateBaseColor, removeBaseColorProcessing]
+  );
+
+  const processMediaBlurhash = useCallback(
+    async (resource) => {
```

Comment:
This should work more like `processMediaBaseColor` above.

The callback itself is not async, but the code in it is wrapped in `(async () => { ... })();`